packages/twenty-website/src/content/developers/self-hosting/self-hosting-var.mdx

```diff
@@ -126,6 +126,7 @@ yarn command:prod cron:calendar:calendar-event-list-fetch
     - Set the following env variables:
       - EMAIL_SMTP_HOST=localhost
       - EMAIL_SMTP_PORT=2525
+      - EMAIL_DRIVER=smtp
```

Comment:
Nice, can you add it also for google and Office configs? lines 108 an 118

Reply:
If so, wouldn't it be better to simply change it in env.example in twenty-server?

Reply:
think we should leave "logger" as default value as it does not require smtp configuration